### PR TITLE
neptune3-ui: replace unit file with drop-in

### PR DESCRIPTION
**Pull request by sashko**
Instead of maintaining own unit file, use the original one provided by
neptune3-ui recipe in meta-boot2qt and append/replace PELUX-specific
configuration with systemd drop-in.

Also put required environment variables into a separate file,
/etc/default/neptune.

Signed-off-by: Oleksandr Kravchuk <oleksandr.kravchuk@pelagicore.com>

### CODE REVIEW

**Comment by JEderonn** [comment]
Have you tried to override dbus-session and neptune to move them to basic.target and see if it will speed up the boot time?

**Comment by sashko** [comment]
> Have you tried to override dbus-session and neptune to move them to basic.target and see if it will speed up the boot time?

Haven't had a chance yet. It's in my TODO list.

File: layers/b2qt/recipes-qt/automotive/neptune3-ui_git.bbappend [hunk 1]
```diff
@@ -6,8 +6,9 @@
 
 FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 
-RDEPENDS_${PN}_append = "\
-    dbus-session \
+SRC_URI += "\
+    file://pelux.conf \
+    file://neptune \
```
Comment (JEderonn) [inline layers/b2qt/recipes-qt/automotive/neptune3-ui_git.bbappend]: Cannot see where/how this file is installed into /etc/defaults/

**Comment by sashko** [comment]
Line 31

File: layers/b2qt/recipes-qt/automotive/neptune3-ui/pelux.conf [hunk 2]
```diff
@@ -0,0 +1,6 @@
+[Service]
+ExecStart=
+ExecStart=/opt/neptune3/neptune3-ui -r --dbus session -c am-config-neptune.yaml -platform eglfs $EXTRA_ARGUMENTS
+WorkingDirectory=/opt/neptune3
```
Comment (JEderonn) [inline layers/b2qt/recipes-qt/automotive/neptune3-ui/pelux.conf]: Is this needed? Already present in https://code.qt.io/cgit/yocto/meta-boot2qt.git/tree/meta-boot2qt/recipes-qt/automotive/neptune3-ui/neptune.service?h=thud

Comment (JEderonn) [reply layers/b2qt/recipes-qt/automotive/neptune3-ui/pelux.conf]: I also wonder if we should replace
`Environment=AM_STARTUP_TIMER=1`
with
`Environment=AM_STARTUP_TIMER=/tmp/am-startup-timing.log`

**Pull request merged**